test/test_torchrun.py

```diff
@@ -0,0 +1,91 @@
+# Owner(s): ["module: unknown"]
+
+from torch.testing._internal.common_utils import TestCase, run_tests
+import os
+import shutil
+import subprocess
+
+class TestTorchrun(TestCase):
+    def test_jemalloc(self):
+        if os.path.isdir('torchrun_logs'):
+            shutil.rmtree('torchrun_logs')
```

Comment:
this can have unintended side-effects depending on the directory where the test is run from (you may end up deleting a legitimate `torchrun_logs` directory). Instead you can use the `setUp()` and `tearDown()` methods and create a temporary test directory for the test that is cleaned up after the test finishes running. Here's an example:

https://github.com/pytorch/pytorch/blob/master/test/distributed/elastic/agent/server/test/local_elastic_agent_test.py#L248-L253

Reply:
done